libs/cardano-data/cardano-data.cabal

```diff
@@ -24,6 +24,7 @@ library
         Data.Universe
         Data.OSet.Strict
         Data.OMap.Strict
+        Data.MonoTuple
```

Comment:
I think you forgot to `git add` the actual module :slightly_smiling_face: 
Happens to me all the time

Reply:
Similarly to `cardano-ledger-core`, this will also mean a minor version bump on `cardano-data` and a lower bound updated for it in `cardano-ledger-shelley`.

Changelog entry will be needed too.